### ISSUE TITLE
Unable to successfully create a working container after Windows 10 Update to 1908

### ISSUE THREAD

**Issue by ivandjordjevic** [Microsoft/nav-docker#318]
Hi,

I have issues setting up my container properly ever since update to Windows 10 1908. I have tried using recent bcsandbox-ltsc2019 image with -updateHosts parameter where I'm able to create a container, run c/side but unable to open web client.

The script I'm using is quite simple and just did a fresh reinstall of docker to 18.09.1 and pulled new image today for 'mcr.microsoft.com/businesscentral/sandbox:us-ltsc2019':
````
$imageName = 'mcr.microsoft.com/businesscentral/sandbox:us-ltsc2019' 
$cred = Get-Credential -UserName $env:USERNAME -Message "Enter your Windows Credentials"
$containerName = 'dev'

New-NavContainer -containerName $containerName `
-imageName $imageName `
-accept_eula `
-Credential $cred `
-auth NavUserPassword `
-licensefile "C:\myfolder\s\mylicense.flf" `
-includecside `
-updateHosts `
-doNotExportObjectsToText  


NavContainerHelper is version 0.4.3.2
Host is Microsoft Windows 10 Pro - ltsc2019
Docker Client Version is 18.09.1
Docker Server Version is 18.09.1
Using image mcr.microsoft.com/businesscentral/sandbox:us-ltsc2019
Removing container dev
Removing dev from hosts
Removing C:\ProgramData\NavContainerHelper\Extensions\dev
Error removing C:\ProgramData\NavContainerHelper\Extensions\dev (attempts: 1), retrying in 10 seconds
Removing C:\ProgramData\NavContainerHelper\Extensions\dev
Creating Nav container dev
Using license file C:\myfolder\s\mylicense.flf
Version: 13.3.27437.0-US
Platform: 13.0.27390.0
Generic Tag: 0.0.9.0
Container OS Version: 10.0.17763.194 (ltsc2019)
Host OS Version: 10.0.17763.253 (ltsc2019)
Using process isolation
Creating container dev from image mcr.microsoft.com/businesscentral/sandbox:us-ltsc2019
a6d888e078d91547b6dbf2c362bc2c97b7f4cc7ba854a05151a3f31317251fe1
Waiting for container dev to be ready
Initializing...
Starting Container
Hostname is dev
PublicDnsName is dev
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Creating Self Signed Certificate
Self Signed Certificate Thumbprint 64F1D89D00103030FF4B9BD759E1DEC2E13200CF
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Using license file 'c:\run\my\license.flf'
Import License
Creating DotNetCore Web Server Instance
Enabling Financials User Experience
Creating http download site
Creating Windows user ivan
Setting SA Password and enabling SA
Creating ivan as SQL User and add to sysadmin
Creating SUPER user
WARNING: The password that you entered does not meet the minimum requirements. 
It should be at least 8 characters long and contain at least one uppercase 
letter, one lowercase letter, and one number.
Container IP Address: 192.168.25.62
Container Hostname  : dev
Container Dns Name  : dev
Web Client          : http://dev/NAV/
Dev. Server         : http://dev
Dev. ServerInstance : NAV

Files:
http://dev:8080/al-2.1.79379.vsix

Initialization took 64 seconds
Ready for connections!
Reading CustomSettings.config from dev
Creating Desktop Shortcuts for dev
Nav container dev successfully created
````
Web client gives me timeout message:
![image](https://user-images.githubusercontent.com/20359384/51707499-ab368f80-2021-11e9-880c-5d57a99b9f76.png)

If I try to download symbols for HelloWorld app I get:
````
[2019-01-24 22:05:35.64] Using reference symbols cache path: c:\Users\Ivan\Documents\AL\ALProject2\./.alpackages
[2019-01-24 22:05:35.65] Please authenticate in the Visual Studio Code...
[2019-01-24 22:05:42.40] Using reference symbols cache path: c:\Users\Ivan\Documents\AL\ALProject2\./.alpackages
[2019-01-24 22:05:42.40] Sending request to http://dev:7049/NAV/dev/packages?publisher=Microsoft&appName=Application&versionText=13.0.0.0
[2019-01-24 22:05:42.41] Sending request to http://dev:7049/NAV/dev/packages?publisher=Microsoft&appName=System&versionText=13.0.0.0
[2019-01-24 22:05:42.41] The request for path /NAV/dev/packages?publisher=Microsoft&appName=System&versionText=13.0.0.0 failed with code ServiceUnavailable. Reason: Service Unavailable
[2019-01-24 22:05:42.41] The request for path /NAV/dev/packages?publisher=Microsoft&appName=Application&versionText=13.0.0.0 failed with code ServiceUnavailable. Reason: Service Unavailable
[2019-01-24 22:05:42.41] Could not download reference symbols. Please ensure that:
1. The correct server name and instance are specified in the launch.json file.
2. The correct application version is specified in the app.json file.
````
With default app.json settings:
````
{
  "id": "8ebea29d-07f1-4d06-abc5-4a6e6bdc4c83",
  "name": "ALProject2",
  "publisher": "Default publisher",
  "brief": "",
  "description": "",
  "version": "1.0.0.0",
  "privacyStatement": "",
  "EULA": "",
  "help": "",
  "url": "",
  "logo": "",
  "capabilities": [],
  "dependencies": [],
  "screenshots": [],
  "platform": "13.0.0.0",
  "application": "13.0.0.0",
  "idRange": {
    "from": 50100,
    "to": 50149
  },
  "runtime": "2.3"
}
````
There seem to be lot of warnings in Event log, not sure which one is relevant. One for example is:

![image](https://user-images.githubusercontent.com/20359384/51708930-6ad91080-2025-11e9-8ff1-094e573fbc38.png)

**Comment by marknitek** [comment]
It think you should/have to use https protocol with NavUserPassword. It created a cert as you can see in the output. But the event you noted states it is not configuerd... Strange
Does https work? 
Side note: always test the webclient in incognito mode if you have any errors.

**Comment by freddydk** [comment]
NavUserPassword works fine with http - the eventlog is just a warning.
If you start a dos prompt and write ping dev - what happens?

**Comment by PleachiM** [comment]
First, check if you get an answer from the ping command like @freddydk mentioned. 
If this is not working try [http://192.168.25.62/NAV](http://192.168.25.62/NAV)

**Comment by freddydk** [comment]
It looks like your browser has HSTS enabled for dev - meaning that it will force https.
You can start your container with -usessl - or you can remove the container and create a container called dev1.
You can also make your browser forget about HSTS for the name - I don't know the details on that.

**Comment by ivandjordjevic** [comment]
Wow, you're right! Trying with container name other than 'dev' worked! I don't know if this got changed since my windows update though which is not that important now. 

I had all my containers (named differently) stopped after windows update to 1908 and after that I started setting up new container using -ltsc2019 images and experimenting with different parameters in container 'dev' without success.

Everything works fine now, even better! Containers are also initiated faster using new images. Thanks a lot!
